public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
+    {
+      "src": "favicon.ico",
+      "sizes": "64x64 32x32 24x24 16x16",
```

Comment:
I forgot to add, but can you empty this array of icons and remove any icons that aren't being used?

Reply:
removed and updated